docs/en-US/component/loading.md

```diff
@@ -92,14 +92,20 @@ If Element Plus is imported entirely, a globally method `$loading` will be regis
 | text        | loading text that displays under the spinner                                                                                                                             | ^[string]                  | —             |
 | spinner     | class name of the custom spinner                                                                                                                                         | ^[string]                  | —             |
 | background  | background color of the mask                                                                                                                                             | ^[string]                  | —             |
-| customClass | custom class name for Loading                                                                                                                                            | ^[string]                  | —             |
+| customClass | custom class name for loading                                                                                                                                            | ^[string]                  | —             |
+| svg         | custom SVG element to override the default loading spinner                                                                                                               | ^[string]                  | —             |
+| svgViewBox  | sets the viewBox attribute for loading svg element                                                                                                                       | ^[string]                  | —             |
+| beforeClose | Function executed before loading attempts to close. If this function returns false, the closing process will be aborted. Otherwise, the loading will close.              | ^[Function]`() => boolean` | —             |
+| closed      | Function triggered after loading has completely closed                                                                                                                   | ^[Function]`() => void`    | —             |
```

Comment:
`beforeClose` and `closed` seem to be ignored in the code. If the documentation is added, the code should also be updated synchronously.

Reply:
I think these methods need to be added, i'll fix this.

Reply:
> `beforeClose` and `closed` seem to be ignored in the code. If the documentation is added, the code should also be updated synchronously.

`directives` no need to add beforeClose & closed. I only added `closed` and `beforeClose` to the declarative call **loading**

Reply:
I don’t know if it was missed before or deliberately not written. Now it is a new addition so it is best to add a new version number.

Reply:
It should be enough to add `beforeClose` and `closed`, and other property configurations will take effect. These two functions will take effect only after this modification.

Reply:
> It should be enough to add `beforeClose` and `closed`, and other property configurations will take effect. These two functions will take effect only after this modification.

ok，i'll fix this.

Reply:
Done. Thank you for patiently responding.